### PR TITLE
DTS-31005: Code cleanup for Auth - not requied

### PR DESCRIPTION
**Pull request by risshukl0**
Removed APIs:

1. private registrationUrl = this.baseUrl + '/api/registerUser';
2.  private standardloginUrl = this.baseUrl + '/api/login';
3. private ldapLoginUrl = this.baseUrl + '/api/ldap';
4. private crowdSsoLoginLoginUrl = this.baseUrl + '/api/login/crowdsso';
5. private forgotPasswordEmailUrl = this.baseUrl + '/api/forgotPassword';
6. private resetPasswordUrl = this.baseUrl + '/api/resetPassword';
7. private getADConfigUrl = this.baseUrl + '/api/activedirectory';

Code cleanup done- removed authenitcation module and AD settings
Fixed the failing 152 test cases, commented 1 remaining test case

### CODE REVIEW

File: UI/src/app/services/app-initializer.service.ts [hunk 1]
```diff
@@ -13,70 +13,66 @@ import { GoogleAnalyticsService } from './google-analytics.service';
 })
 export class AppInitializerService {
 
-  constructor(private sharedService: SharedService, private httpService: HttpService, private router: Router, private featureToggleService: FeatureFlagsService, private http: HttpClient, private route: ActivatedRoute, private ga: GoogleAnalyticsService) { 
-    this.checkFeatureFlag(); 
-    this.validateToken(); 
+  constructor(private sharedService: SharedService, private httpService: HttpService, private router: Router, private featureToggleService: FeatureFlagsService, private http: HttpClient, private route: ActivatedRoute, private ga: GoogleAnalyticsService) {
+    this.checkFeatureFlag();
+    // this.validateToken();
```
Comment (HinPublicis) [inline UI/src/app/services/app-initializer.service.ts:18]: @risshukl0 why did you comment this?

File: UI/src/app/services/sso.guard.ts [hunk 2]
```diff
@@ -17,15 +17,8 @@ export class SSOGuard implements CanActivate {
...
-    if (!environment.SSO_LOGIN) {
-      if (!environment.AUTHENTICATION_SERVICE) {
-        return true;
-      } else {
-        return false;
-      }
-    } else {
-      return this.getSSOUserInfo();
-    }
+    return this.getSSOUserInfo();
```
Comment (HinPublicis) [inline UI/src/app/services/sso.guard.ts:20]: @risshukl0 we will require environment.SSO_LOGIN check here

**Review by HinPublicis — CHANGES_REQUESTED** [review]

File: UI/src/app/services/app-initializer.service.ts (hunk 1 above)
Comment (risshukl0) [reply UI/src/app/services/app-initializer.service.ts:18]: I don't remember removing this function call. Might have done by mistake. Restored.

File: UI/src/app/services/sso.guard.ts (hunk 2 above)
Comment (risshukl0) [reply UI/src/app/services/sso.guard.ts]: Fixed. Taken code from master.

**Pull request closed without merging**